app/models/application_user_role.py

```diff
@@ -0,0 +1,31 @@
+import enum
+from models import db
+from sqlalchemy import (
+    Column, Integer,
+    Enum,
+    ForeignKey,
+)
+from sqlalchemy.orm import relationship
+
+
+class Role(enum.Enum):
+    view = 1
+    edit = 2
+    admin = 3
```

Comment:
I am not familiar with the role definition. Are `view`, `edit` and `admin` common words for user role?

Reply:
hmm, how about `viewer` / `editor` / `owner` ?
https://cloud.google.com/iam/docs/understanding-roles
I'll rename.